### DISCUSSION TITLE
How to make VSCode recognize TailwindCSS syntax?

### DISCUSSION THREAD

**Discussion by eriandev** [tailwindlabs/tailwindcss#2486] [Help] [answered]
![image](https://user-images.githubusercontent.com/4275190/94997515-3a26d980-0571-11eb-8f01-72422271344a.png)

**Comment by simonswiss** [comment, marked as answer]
Hey!

I assume you are getting the `semi-colon expected` error:

![image](https://user-images.githubusercontent.com/485747/95046536-13fb6800-0730-11eb-8c34-827fcb689652.png)

You probably want to add the [PostCSS language support](https://marketplace.visualstudio.com/items?itemName=csstools.postcss) extension to vscode.

You can then select `PostCSS` as the language for your CSS file (if not done automatically), so it stops trying to warn you about CSS syntax errors.

![image](https://user-images.githubusercontent.com/485747/95046301-a5b6a580-072f-11eb-8432-2f02d138b39d.png)

Hope it helps! 👍

**Reply by lights0123** [reply]
Same error. Is there a way to do it with scss?

**Reply by ttntm** [reply]
Thanks for that, been ignoring CSS files with "errors" for way too long and decided to look it up today which led me here.

**Reply by FairyEver** [reply]
<img width="500" alt="Xnip2021-04-27_22-43-59" src="https://user-images.githubusercontent.com/19357920/116261293-13005580-a7aa-11eb-87c7-448df2f506eb.png">

How to solve this problem？in `<style lang="scss">`
